src/app/app.ts

```diff
@@ -77,7 +77,7 @@ class App {
   }
 
   // из-за этого багался колл-стек, разобрать.
```

Comment:
Add TODO into the all comments. It will help you to find them and do not forget about them.